Automattic/jetpack
# Social: Mention Manual Sharing in connection management

Fixes https://github.com/Automattic/jetpack-reach/issues/464

## Proposed changes:

* Added mention into the bottom of the connection management modal, and to not supported connections.

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/464
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Go to the admin page.
* Open the connection management modal, and check that you see the text and link in the bottom.
* Have a broken connection. In `projects/js-packages/publicize-components/src/components/connection-management/connection-status.tsx` set service to `null` to simulate an unsupported connection.
* Check that other than saying it's unsupported, we tell them to use Manual Sharing

<table>
    <tbody>
        <tr>
            <th>Connection list</th>
        </tr>
        <tr>
            <td><img alt="CleanShot 2024-07-18 at 15 28 55 png" src="https://github.com/user-attachments/assets/00dbfad0-53a4-41a6-babb-5a854ec1db9a"></td>
        </tr>
        <tr>
            <th>Modal</th>
        </tr>
        <tr>
            <td><img alt="CleanShot 2024-07-18 at 15 29 04 png" src="https://github.com/user-attachments/assets/370645f1-63af-4d76-8be0-39724785231f"></td>
        </tr>
        <tr>
            <th>Unsupported connection in modal</th>
        </tr>
        <tr>
            <td><img alt="CleanShot 2024-07-18 at 15 29 10 png" src="https://github.com/user-attachments/assets/56e10b3c-9a58-4ec9-99ca-90567af4834d"></td>
        </tr>
    </tbody>
</table>
